lib/matplotlib/offsetbox.py

```diff
@@ -963,13 +966,18 @@ def get_extent(self, renderer):
         mtx = self.offset_transform.matrix_from_values(*_off)
         self.offset_transform.set_matrix(mtx)
 
-        return ub.width, ub.height, 0., 0.
+        dpi_cor = renderer.points_to_pixels(1.)
+        return ub.width*dpi_cor, ub.height*dpi_cor, 0., 0.
```

Comment:
I think pep8 is fine with that for arithmetic operators, it calls for consistency,  readability/good judgment. Either way, when I make a revision to the PR that line will not be there(for a yet known reason).